automatic/waterfox/waterfox.nuspec

```diff
@@ -14,19 +14,16 @@
     <licenseUrl>https://github.com/MrAlex94/Waterfox/blob/master/LICENSE.md</licenseUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <projectSourceUrl>https://github.com/MrAlex94/Waterfox</projectSourceUrl>
-    <tags>waterfox admin web-browser firefox 64-bit</tags>
+    <tags>waterfox web-browser firefox 64-bit admin cross-platform foss</tags>
     <summary>Waterfox is basically a 64-bit version of Firefox.</summary>
     <description>
-## How and why Waterfox is here...
-Waterfox was started back in March 2011 by myself ([Alex Kontos](https://github.com/MrAlex94)), a 16 year old student.
-I had a fascination for the web and wanted to help expand on the ideals of what Mozilla had for a free and open web.
-And so I decided to make Waterfox, a 64-Bit browser based on Mozilla's free and open source platform.
+Waterfox was started back in March 2011 by [Alex Kontos](https://github.com/MrAlex94), a 16 year old student.
+He had a fascination for the web and wanted to help expand on the ideals of what Mozilla had for a free and open web.
+And so he decided to make Waterfox, a 64-Bit browser based on Mozilla's free and open source platform.
 
 Waterfox was one of the first widely distributed 64-Bit browsers on the web and quickly gained a loyal following.
 At a time Waterfox had one thing in mind: speed, but now Waterfox also attempts to be an ethical, user-oriented browser
 
-![](https://www.waterfoxproject.org/img/storm-speed.png)
```

Comment:
Same as above (here it looks even more out of place, as alignment is different than on the waterfox page, and also background color doesn't match)